src/ducks/user.duck.js

```diff
@@ -301,13 +315,13 @@ export const fetchCurrentUser = () => (dispatch, getState, sdk) => {
     return Promise.resolve({});
   }
 
-  const params = {
+  const parameters = params || {
```

Comment:
Did I understand this correctly that by default `profileImage` and `stripeAccount` are included but if a different relationship is passed to the `fetchCurrentUser` then these defaults are not included?

Reply:
@OtterleyW yes. I was thinking of creating some kind of extra params pattern, but since currentUser is always fetched if the user is authenticated, I decided that `mergeCurrentUser` function will handle those cases. `mergeCurrentUser` is needed anyway, otherwise, someone would need to keep track that which relationships need to be included by different pages.

So, `fetchCurrentUser({ include: ['stripeCustomer.defaultPaymentMethod'] });` will add `stripeCustomer` to the previously fetched `currentUser` entity.

If there's a need to ensure that profileImage and stripeAccount are included for the page component, then it's better to call all of them 
```js
fetchCurrentUser({ 
  include: ['stripeCustomer.defaultPaymentMethod', 'profileImage', 'stripeAccount'],
  'fields.image': ['variants.square-small', 'variants.square-small2x'] });
```